src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx

```diff
@@ -115,6 +138,10 @@ function LineNumberTooltip({ editor, keyModifiers }: Props) {
       }
     }, 10);
 
+    const debouncedFetchHitCountsForVisibleLines = debounce(() => {
+      fetchHitCountsForVisibleLines();
+    }, 100);
```

Comment:
I'm curious if you can feel the tenth of a second lag. I feel like maybe this should be pretty low, just to take care of the *really* fast scrolling case. Having it be responsive as you scroll feels really good if we can pull it off. Anyways - we can iterate on this as well.

Reply:
I tried 10ms initially, and that wasn't enough time - this was still getting called a bunch. I wanted to make sure we wouldn't keep calling immediately as you move the mouse up and down lines.